src/Makefile

```diff
@@ -3,17 +3,25 @@ PLATFORM_DIR = platforms/$(PROBE_HOST)
 VPATH += $(PLATFORM_DIR) target
 ENABLE_DEBUG ?=
 
+CC ?= gcc
```

Comment:
CC is already set in the platforms and needs to use the form `$(CROSS_COMPILE)gcc`. We are fine with the assignment being moved into this Makefile, but it must not semantically change in its form for now.

Reply:
when checking I realized there was no need to assign CC in the Makefile as it is already set - so I've removed from the proposed patch